src/com/sibvisions/kitchensink/samples/containers/SplitPanelSample.java

```diff
@@ -74,4 +118,25 @@ public String getName()
 		return "Split";
 	}
 	
+	/* Creates an {@link ICellEditor} that can be used for editing the

```

Comment:
Javadoc starts with `/**` like this:

    /**
     * Text goes here.
     *
     * @param blabla
     */